src/components/LetterBtn/letterBtn.module.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../styles/colors.scss';
```

Comment:
We can use an import alias for this:

```css
@import '@/styles/colors.scss';
```

Reply:
ach cool. didn't know that. corrected it in all scss files (u used absolute path in the footer too, I just copy you :D